src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs

```diff
@@ -1079,5 +1079,10 @@ private void OnGeometryScaling_Click(object sender, RoutedEventArgs e)
             }
             GeoScalingPopup.IsOpen = true;
         }
+
+        private void InCanvasSearchBar_Opened(object sender, EventArgs e)
+        {
+            int i = 0;
+        }
```

Comment:
This is not meant to be committed right?

Reply:
@QilongTang code removed in the next commit: https://github.com/DynamoDS/Dynamo/pull/13794/commits/e3416c6dbc972187f66a46f9243770f360d2c144